var/spack/repos/builtin/packages/py-rx/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyRx(PythonPackage):
+    """Reactive Extensions (Rx) for Python"""
+
+    homepage = "http://reactivex.io/"
+    pypi     = "Rx/Rx-3.2.0.tar.gz"
+
+    maintainers = ['dorton21']
+
+    version('3.2.0', sha256='b657ca2b45aa485da2f7dcfd09fac2e554f7ac51ff3c2f8f2ff962ecd963d91c')
+    version('1.6.1', sha256='13a1d8d9e252625c173dc795471e614eadfe1cf40ffc684e08b8fff0d9748c23')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-pytest')
+    depends_on('py-coveralls')
+    depends_on('py-flake8')
```

Comment:
These are only needed for testing, but Spack doesn't run pytest, so these deps can be removed.